tests/sidetrail/working/stubs/s2n_hash.c

```diff
@@ -43,6 +43,11 @@ int s2n_hash_new(struct s2n_hash_state *state)
   return SUCCESS;
 }
 
+S2N_RESULT s2n_hash_state_validate(struct s2n_hash_state *state)
+{
+    return S2N_RESULT_OK;
+}
```

Comment:
@danielsn I wrote these stubs following the same pattern as the ones for `s2n_hash_new` functions.

Reply:
LGTM